backend/controllers/user.controller.ts

```diff
@@ -65,4 +65,47 @@ export const update = async (req: Request, res: Response) => {
   }
 }
 
+export const deactivate = async (req: Request, res: Response) => {
+  const userId = { _id: req.params.id };
+  const update = { active: false }
+  try {
+    await User.findOneAndUpdate(userId, update);
+
+    res.json({ ok: true, message: "User deactivated successfully" })
+  } catch (error) {
+    res.status(StatusCode.INTERNAL_SERVER_ERROR).json({ message: error.message });
+  }
+}
+
+
+export const deleteOne = async (req: Request, res: Response) => {
+  const userId = { _id: req.params.id }
+
+  try {
+    const user = await User.findOne(userId)
+    if (!user) {
+      return res.status(StatusCode.NOT_FOUND).json({ message: 'User not found' });
+    }
+
+
+    const userBoss = user.boss;
+
+    if (user.boss === null) {
```

Comment:
```suggestion
    if (!userBoss) {
```
Please be consistent and use the same syntax